public/gameInfo.php

```diff
@@ -1243,25 +1231,46 @@ function ResetProgress() {
                             );
                             echo " <span class='TrueRatio'>($achTrueRatio)</span>";
                             echo "</div>";
-                            echo "<div class='mb-2'>$achDesc</div>";
+                            echo "<div>$achDesc</div>";
                             if ($achieved) {
-                                echo "<div class='date smalltext'>Unlocked $dateAch</div>";
+                                echo "<div class='date smalltext italic'>Unlocked on $dateAch</div>";
```

Comment:
Italics should be used to emphasize something. The fact that this is already grey and small suggests it's not important and should not be emphasized.